packages/components/Tabs/src/TabsItem.settings.windows.ts

```diff
@@ -2,6 +2,8 @@ import { tabsItemName, TabsItemType } from './TabsItem.types';
 import { IComposeSettings } from '@uifabricshared/foundation-compose';
 import type { IViewProps } from '@fluentui-react-native/adapters';
 
+export const tabsItemSelectActionLabel = 'Select a TabsItem';
+
```

Comment:
Where is this used? A hardcoded English string would not be good for internationalization if it's shipping with the control...

Reply:
Note: this is already in other platform forks, so this is a bug on us